packages/jni-swig-stub/build.gradle.kts

```diff
@@ -62,13 +72,6 @@ realmPublish {
     }
 }
 
-java {
-    withSourcesJar()
-    withJavadocJar()
```

Comment:
I am not sure that we can remove this. There are some subtleties about publishing to Maven central that requires some of these. But maybe it is only the sources - Let's see when we release 🙈

Reply:
I have added them back.